ch03.asciidoc

```diff
@@ -556,3 +556,33 @@ When the channel is closed, the final balance of the channel is submitted to the
 For instance: if the opening balance was 1 BTC to Alice, and Alice made a payment of 0.3 BTC to Bob, then the final balance of the channel is 0.7 BTC to Alice and 0.3 BTC to Bob.
 If Alice tries to cheat by submitting the opening state of the channel to the Bitcoin blockchain, with 1 BTC to Alice and 0 BTC to Bob, then Bob can retaliate by submitting the true final state of the channel, as well as penalty transaction that gives him all Bitcoin in the channel.
 In this context, the Bitcoin blockchain acts as a court system; recording the initial and final balances of each channel, and approving penalties if one of the parties tries to cheat.
+
+
+=== Commonality of Bitcoin and Lightning
+
+==== Monetary Unit
+
+Both the Bitcoin network and the Lightning network use the same monetary units: bitcoins. 
+Lightning payments use the very same bitcoins as Bitcoin transactions.
+
+==== Irrevocability 
+
+Both, Bitcoin transactions and Lightning payments are irrevocable. 
+There is no "undo" operation for either one. 
+As a user of either one you have to act responsibly.
+
+=== Trustlessness
+
+Just as Bitcoin, Lightning requires the user only to trust mathematics, encryption and that the Lightning code does not have any critical bugs.
```

Comment:
Lot of missing nuance here...

As an alternative, we could say something along that lines of "the security model of Lightning reduces to the security of the base blockchain.

Reply:
Fair and correct. Your suggestion is worked in now.

See Commit # 3